packages/scss/src/commons/core.scss

```diff
@@ -13,8 +13,10 @@ $gaps: 'column-gap', 'row-gap', 'gap';
 $displays: 'block', 'flex', 'inline', 'inline-flex', 'inline-block', 'contents', 'none';
 $direction: 'column', 'column-reverse', 'row', 'row-reverse';
 $wrap: 'wrap', 'nowrap', 'wrap-reverse';
-$justify: 'flex-start', 'flex-end', 'space-around', 'space-between', 'space-evenly', 'center';
-$align: 'flex-start', 'flex-end', 'baseline', 'stretch', 'center';
+// flex-start and flex-end are deprecated
```

Comment:
They are not really deprecated, just specific to flex boxes. `start` will default to `flex-start` in the context of a flex container, but `flex-start` will not always give the same result as `start` in every context.

![image](https://github.com/user-attachments/assets/2a753419-4758-48da-ab8d-941b057af2d1)

![image](https://github.com/user-attachments/assets/8f0f9373-261e-4a85-87c3-192a6fe6675c)